appveyor.yml

```diff
@@ -50,5 +50,6 @@ build_script:
     - cd "%APPVEYOR_BUILD_FOLDER%"
     - mkdir NotepadTest\unicode\plugins
     - mkdir NotepadTest\unicode\updater
+    - rename paths.props.appveyor paths.props
```

Comment:
Good point, probably there will be no commit on the appveyor build, but staying in a clean mode is the better way.